.github/workflows/pnp-service-definition-validate.yml

```diff
@@ -0,0 +1,39 @@
+name: Validate Service Definition File(s)
+on:
+  workflow_call:
+    inputs:
+      service-type:
+        description: 'Type of the service.'
+        type: string
+        required: true
+      service-definitions:
+        description: 'List of service definition file paths provided as a JSON array string. For example: ["./folder1/service-definition1.yaml", "./folder2/service-definition2"].'
+        type: string
+        required: true
+
+jobs:
+  validate-service-definition:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Configure Git
+        uses: extenda/actions/setup-git@v0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Fetch latest changes
+        run: git pull origin ${{ github.ref }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Validate service definition file(s)
+        uses: extenda/shared-workflows/composite-actions/nodejs-script/service-definition-validator@feat/EA-2490_createActionForServiceDefinitionFilesValidation
```

Comment:
Will you change the branch to the `master` later on?

Reply:
Good point. Have set it to this branch for testing purposes but seems forgot to revert.